ceph-salt-formula/salt/ceph-salt/apply/tuned-latency.sls

```diff
@@ -0,0 +1,29 @@
+
+{% if grains['id'] in pillar.get('ceph-salt:minions:latency', []) %}
```

Comment:
Instead of accessing both `grains` and `pillar`, and for consistency with other stages you can simply access `grains`:

```
{% if 'latency' in grains['ceph-salt']['roles'] %}
```